src/htmlgen.cpp

```diff
@@ -1769,7 +1769,7 @@ void HtmlGenerator::writeStartAnnoItem(const QCString &,const QCString &f,
   if (!path.isEmpty()) docify(path);
   QCString fn = f;
   addHtmlExtensionIfMissing(fn);
-  m_t << "<a class=\"el\" href=\"" << fn << "\">";
+  m_t << "<a class=\"RENAMED2_el\" href=\"" << fn << "\">";
```

Comment:
Probably based on some old and incorrect changes for some other issues you had, to be reverted